### DISCUSSION TITLE
v2.8.10

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4723] [分享 / Share]
<details>
<summary>获得 v2.8.9-preview 的所有新功能, 点击展开查看</summary>

- `下载视频 - WASM 混流输出` 支持更持久的缓存方式. (PR #4667 by [WakelessSloth56](https://github.com/WakelessSloth56))
- `外置稍后再看` 和 `启用快速收藏` 增加了 `显示方式` 选项, 出现无法自动适配的排版问题时可以尝试手动调整此设置. (#4532)
- 新增插件 `动态过滤器 - 移除商品带货动态`. (#4425)
> 移除动态里的商品带货动态 (UP主的推荐 · 来自 XX), 装有 `动态过滤器` 时生效.

- 新增组件 `动态图片平铺展示`. (#4645)
> 将动态中左右切换式的图片改回传统的平铺展示. (在动态详情中可能稍有延迟)

- 更换了 `自定义顶栏` 的动态提醒数字来源. (#4427)

</details>

✨新增
- 夜间模式更新对视频和直播页面的样式. (#4701, PR #4675 by [WangYK](https://github.com/AnotiaWang))
- 增强 `下载视频 - WASM 混流输出` 对错误输入流的检测, 支持 FLAC 音频流 (产生 `.mkv` 文件). (PR #4686 by [DBeidachazi](https://github.com/DBeidachazi))
- `复制动态链接` 改用 `opus` 格式的链接, 并支持新版话题页面. (PR #4695, PR #4696 by [星海](https://github.com/lovelyCARDINAL))
- 同时安装了 pakku 时, 支持下载处理后的弹幕. (PR #4712 by [xmcp](https://github.com/xmcp))
- `传统连播模式` 支持识别和处理视频列表. (#4699, PR #4713 by [Wuch](https://github.com/IEXPECTATION))
- `删除视频弹窗` 支持屏蔽收起后的弹窗. (#4703)

🐛修复
- 再次修复 `快速收起评论` 遮挡评论的表情等弹窗.
- 修复评论区相关功能失效. (#4690)
- 修复 `自定义顶栏` 导致稍后再看页面布局错位. (#4668)

☕开发者相关
- Comment API 现已重构并支持 Vue 3 版本的评论区. (#4690)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.10'>v2.8.10</a>.</em>

**Comment by Kal-tsit63** [comment]
想问一下，这次更新以后，评论区显示IP就正常了吗？
我这里更新完成并检查插件更新以后还是没办法显示

**Reply by the1812** [reply]
那个还没有, 见 #4643
